docs/conf/eu/2017/news/announcing-presentations-ticket-update.rst

```diff
@@ -0,0 +1,63 @@
+
+The topics this year cover a vast range of subjects, points of view, and real-world case studies. We hope that whatever your role may be, you can find something to relate to.
+
+Following the success of the Portland conference, we also ran a CFP for unconference sessions, so we could have a handful of committed "anchor sessions" to help kick off the unconference.
+Of course you can still sign up to run an unconference session during the conference, but we hope that having a few scheduled sessions will help you with your personal schedule planning.
+
+You'll find more info and links to the full talk abstracts down below. You can also visit the `Sessions <http://www.writethedocs.org/conf/eu/2017/speakers/>`_ page for the full list of presentations.
+
+Presentations
+-------------
+
+* Becky Todd - `A content manager's guide to crowdsourcing the docs </conf/eu/2017/speakers/#speaker-becky-todd>`_
+* Chris Ward - `Documentation beyond words </conf/eu/2017/speakers/#speaker-chris-ward>`_
+* Christy Lutz - `You have already succeeded: Design critique guidelines make feedback easier </conf/eu/2017/speakers/#speaker-christy-lutz>`_
+* Cory Williamson-Cardneau - `Sticks & Stones... Microaggressions & Inclusive Language at Work </conf/eu/2017/speakers/#speaker-cory-williamson-cardneau>`_
+* Daniel D. Beck - `Deprecate and destroy: documenting your software’s last days </conf/eu/2017/speakers/#speaker-daniel-d-beck>`_
+* Daniele Procida - `The four kinds of documentation, and why you need to understand what they are </conf/eu/2017/speakers/#speaker-daniele-procida>`_
+* Garen Torikian - `Finally! Trustworthy and Sensible API Documentation with GraphQL  </conf/eu/2017/speakers/#speaker-garen-torikian>`_
+* Kate Wilcox - `Aw Snap!  The Docs, They Are A-Changin’ (with apologies to Bob Dylan) </conf/eu/2017/speakers/#speaker-kate-wilcox>`_
+* Lauri Apple - `Telling a Great Story on GitHub </conf/eu/2017/speakers/#speaker-lauri-apple>`_
+* Lesia Zasadna - `Requirements that you didn't know were there </conf/eu/2017/speakers/#speaker-lesia-zasadna>`_
+* Meike Chabowski - `An Alien Looking From the Outside In: Main Takeaways After One Year in Documentation </conf/eu/2017/speakers/#speaker-meike-chabowski>`_
+* Raphael Pierzina - `Hi, my name is README! </conf/eu/2017/speakers/#speaker-raphael-pierzina>`_
+* Ruthie BenDor - `Even Naming This Talk Is Hard </conf/eu/2017/speakers/#speaker-ruthie-bendor>`_
+* Stuart Culshaw - `Tech Writers Without Borders: Making the world a better place, one (numbered) step at a time </conf/eu/2017/speakers/#speaker-stuart-culshaw>`_
+* Thomas Parisot - `Writing a book in 2017  </conf/eu/2017/speakers/#speaker-thomas-parisot>`_
+* Tim Rogers - `"You never get a second chance to make a first impression": writing great "getting started" documentation </conf/eu/2017/speakers/#speaker-tim-rogers>`_
```

Comment:
We have a backup but it's basically one talk less and we wanted to have an earlier finish to day 2 so it works out I think.